README.md

```diff
@@ -43,6 +43,23 @@ func _ready():
 	player.play(LPCAnimatedSprite2D.LPCAnimation.WALK_UP)
 ```
 
+## Support for Oversized Weapons
+Just replace the the LPCSpriteSheet with the OversizedLPCSpriteSheet on the panel and use it same as LPCSpriteSheet
+
+Currently the animations supported are:
+ - SLASH_UP
+ - SLASH_LEFT
+ - SLASH_DOWN
+ - SLASH_RIGHT
+
+You will need 4 sprites for the oversized weapons to work properly:
```

Comment:
As completed sprites (such as those made using [the generator](https://sanderfrenken.github.io/Universal-LPC-Spritesheet-Character-Generator/#?body=Body_color_light&head=Human_male_light)) don't need any adjustments, might I suggest:

"If using component sprites rather than a a pre-built spritesheet, pay attention to the order in which they are configured as it will influence visibility."

Reply:
May add that pre-built spritesheets for oversized weapons are not supported yet